website/docs/plugins/ax_sweeper.md

```diff
@@ -70,15 +71,15 @@ banana_y:
  bounds: [-5, 10.1]
 ```
 
-The `x` parameter takes on a "range" of integer values, between `-5` to `5`. The `y` parameter takes on a range of floating-point values between `-5` to `10.1`. In general, the plugin supports all the [Parameters](https://ax.dev/api/core.html?highlight=range#module-ax.core.parameter) that Ax supports. According to the [Ax documentation](https://ax.dev/api/service.html#ax.service.ax_client.AxClient.create_experiment), the required elements in the config are:
+The `x` parameter takes on a "range" of integer values, in the interval `[-5, 5]`. The `y` parameter takes on a range of floating-point values in the interval `[-5, 10.1]`. In general, the plugin supports all the [Parameters](https://ax.dev/api/core.html?highlight=range#module-ax.core.parameter) that Ax supports. According to the [Ax documentation](https://ax.dev/api/service.html#ax.service.ax_client.AxClient.create_experiment), the required elements in the config are:
```

Comment:
The `x` parameter is in the integer range -5 to 5. `interval` is a keyword and it always means floating point.